app/assets/stylesheets/partials/custom.scss

```diff
@@ -132,30 +133,30 @@ a:active {
 }
 
 .carousel-indicators li {
-display: inline-block !important;
-width: 10px !important;
-height: 10px !important;
-margin: 1px !important;
-text-indent: -999px !important;
-border: 1px solid #fff !important;
-border-radius: 10px !important;
-cursor: pointer !important;
-background-color: #000 \9 !important;
-background-color: rgba(0,0,0,0) !important;
+  display: inline-block !important;
```

Comment:
!important should not be used<br>Properties should be ordered background-color, background-color, border, border-radius, cursor, display, height, margin, text-indent, width